packages/opentelemetry-types/tsconfig.json

```diff
@@ -6,5 +6,11 @@
   },
   "include": [
     "src/**/*.ts"
-  ]
+  ],
+  "typedocOptions": {
+    "name": "OpenTelemetry Documentation",
+    "out": "docs",
```

Comment:
There could be other documentation in the `docs` folder such as markdown files. The output for Typedoc should probably be a subfolder or simply a completely different folder.

Reply:
Changed `out` to documentation folder. PTAL

Reply:
This issue still exists as `documentation` is equivalent to `docs`. There should never be both folders at the same time since this would be very confusing. Maybe something like `docs/out` or `build/docs`?

Reply:
Sure